kustomize/argo3/base/argo.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    ksonnet.io/component: mm-argo
+  name: mm-argo-binding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: dkube-clusterrole
+subjects:
+- kind: ServiceAccount
+  name: modelmonitor
+  namespace: modelmonitor
+---
+apiVersion: v1
+data:
+  config: |
+    artifactRepository:
+      archiveLogs: false
+    instanceID: modelmonitor
+kind: ConfigMap
+metadata:
+  labels:
+    ksonnet.io/component: mm-argo
+  name: mm-workflow-controller-configmap
+  namespace: modelmonitor
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    ksonnet.io/component: mm-argo
+  name: mm-workflow-controller
+  namespace: modelmonitor
+spec:
+  selector:
+    matchLabels:
+      app: mm-workflow-controller
+  template:
+    metadata:
+      labels:
+        app: mm-workflow-controller
+    spec:
+      containers:
+      - args:
+        - --configmap
+        - mm-workflow-controller-configmap
+        - --executor-image
+        - argoproj/argoexec:v3.1.13
+        - --namespaced
+        command:
+        - workflow-controller
+        env:
+        - name: LEADER_ELECTION_IDENTITY
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.name
+        image: docker.io/argoproj/workflow-controller:v3.1.13
+        livenessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /healthz
+            port: 6060
+          initialDelaySeconds: 90
+          periodSeconds: 60
+          timeoutSeconds: 30
+        name: mm-workflow-controller
+        ports:
+        - containerPort: 9090
+          name: metrics
+        - containerPort: 6060
+        securityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop:
+            - ALL
+          readOnlyRootFilesystem: true
+          runAsNonRoot: true
+      securityContext:
+        runAsNonRoot: true
+      dnsPolicy:  ClusterFirst
+      schedulerName: default-scheduler
+      nodeSelector: NODE_SELECTOR
```

Comment:
No harm in keeping the 'nodeSelector' at line 85?

Reply:
https://github.com/oneconvergence/gpuaas/blob/e40f842972fbd37af70d03ac0d3a80bc35b704db/dkube/go/src/installer/d3/installer.go#L2742 we have for every resource this nodeSelector logic in installer.go. So i think it might be safe. @mak-454 can you once verify